lasagne/layers/conv.py

```diff
@@ -249,7 +249,7 @@ def get_output_for(self, input, input_shape=None, **kwargs):
                                       filter_shape=self.get_W_shape(),
                                       border_mode='full')
             shift = (self.filter_size[0] - 1) // 2
-            conved = conved[:, :, shift:input.shape[2] + shift]
+            conved = conved[:, :, shift:-shift]
```

Comment:
I haven't done the math, but I'm unsure whether this amounts to the same thing for every possible length. Have you? I wonder if there is maybe a reason that I wrote `input.shape[2] + shift` instead of `-shift` in the first place (but there might not be).